tests/unit/metadata/bucketclient/backend.js

```diff
@@ -0,0 +1,92 @@
+const assert = require('assert');
+const http = require('http');
+const BucketClientInterface =
+    require('../../../../lib/metadata/bucketclient/backend').default;
+
+const bucketName = 'test';
+const invalidBucket = 'invalid';
+const params = {};
+const logger = {
```

Comment:
You can use the `DummyRequestLogger` class to fake the log for your unit test.

Reply:
Thank you. I have looked at the `DummyRequestLogger` helper and will perform the necessary changes.